i18n.zh-cn.md

```diff
@@ -0,0 +1,315 @@
+# i18n
+
+This sketch contains the survey text that will be presented to non-English speakers.
+
+## Welcome to the PureScript Survey! 欢迎参加PureScript社区调查
+
+We're delighted that you've taken the time to answer our first community survey. This survey will help us focus our efforts as a community on the issues that matter most to our members. If you're not yet part of the PureScript community, we'd still love to hear from you, as we are always looking to improve our onboarding process and are eager to establish links with new communities.
+
+非常感谢你能抽出时间参与我们第一次社区调查！这次调查能帮助我们集中精力来解决社区最关心的问题。此外我们也一直致力于改进新用户加入社区的流程，并且渴望和新社区建立联系。所以即使你还不是PureScript社区成员之一，我们也希望听到你的声音。
+
+### Do you use PureScript? 你现在使用PureScript吗？
+
+Select one. 请选择一个最合适的描述。
+
+- Yes, I use PureScript (for any purpose, even if you're just learning).
+  是的，我现在使用PureScript （不限目的，学习PureScript也包括在内）。
+- No, I don't currently use PureScript, but I have in the past.
+  不，我现在并不使用PureScript。但是以前用过。
+- No, I have never used PureScript.
+  不，我从未用过PureScript。
+
+## For previous PureScript users 过去使用过PureScript的用户
+
+### As you have indicated that you're no longer using PureScript, what prompted you to participate in this survey? 在你已经不使用PureScript的情况下，是什么原因驱使你参与这次调查？
+
+Select all that apply. 请选择所有符合的选项。
+
+- I plan to return to using PureScript in the future.
+  我计划在未来继续使用PureScript。
+- I consider myself part of the PureScript community.
+  我认为我是PureScript社区的一员。
+- Specifically to provide feedback on why I stopped using PureScript.
+  我希望反馈停止使用PureScript的原因。
+- To provide feedback on PureScript in general.
+  我希望反馈建议。
+- Curiosity.
+  好奇心
+- Other (open response)
+  其他（请注明）
+
+### What issues in the PureScript ecosystem do you feel need the most urgent attention? 你觉得在PureScript生态中，哪些是最亟待解决的问题？
+
+Select all that apply. 请选择所有符合的选项。
+
+- The community is not large enough.
+  社区太小了。
+- There are not enough large companies using PureScript in production.
+  在生产环境中使用PureScript的大公司太少了。
+- There is not yet a "killer app" written in PureScript.
```

Comment:
Here I have a question about `app`. Does it stand for application as in "mobile app", or application as "solution to a set of problems"?

Reply:
Application as in "mobile app"